### ISSUE TITLE
Help with subtrees

### ISSUE THREAD

**Issue by podefr** [nodejs/iojs-fr#77]
Hey,

@snostorm @tdd @yoannmoinet I'm calling the subtrees experts

I'm struggling with the subtrees thing we're trying to introduce. We've merged a couple of PRs recently into the `website` branch and I don't understand how to notify the `iojs.org` repository so that it gets the latest updates.

Here are the last two pull requests that we've merged into the `website` branch:
https://github.com/nodejs/iojs-fr/pull/73
https://github.com/nodejs/iojs-fr/pull/74

I noticed that we have a `website` branch that isn't listed on the list of branches on this `iojs-fr` repository, and a `iojs-fr_website_subtree` branch that I can't checkout locally. Why is that? Is the ìojs-fr_website_subtree` branch designed to receive the latest changes or was created for something else?

Also, it seems that the commands that @snostorm mentioned should be executed on `iojs.org` for which we don't have write permissions:

```bash
git fetch iojs-fr website
git pull --squash -s subtree iojs-fr website
git commit -m "Updates iojs-fr content from iojs/iojs-fr#website"
```

What am I missing?

Thanks,
Olivier

**Comment by yoannmoinet** [comment]
I've also started the process like 2 weeks ago. And got messed up with the steps, I think I'm the one that created the `iojs-fr_website_subtree` branch.

I think we should wait for @snostorm to see what he would do with our branchs precisely, that would help to understand the steps.

And indeed these commands are to be executed on a `iojs.org` fork of your own, then, this will be your pull-request over there. (I think)

**Comment by podefr** [comment]
yeah the PR bit makes sense to get our changes in, but then what are the benefits of using subtrees? still confused :)

**Comment by yoannmoinet** [comment]
I'm still a little confused too :smile: but I thought it was essentially to have a 'persistent' link between the two. And be able to modify ours to then push to theirs.

**Comment by snostorm** [comment]
Hey, I'm formulating a better answer and digging in to the two branches in question.

Initial response:
- The subtree branch is designed to keep an area in this repo 100% mapped to the **[fr](https://github.com/nodejs/iojs.org/tree/master/content/fr)** folder of the https://github.com/nodejs/iojs.org project (and later one for the new node.js website project) without needing to figure what is/isn't out of date between different file structures.
- ^ This allows for internal https://github.com/nodejs/iojs-fr PR review process on the subtree branch which, when ready, merges and can be used to build an https://github.com/nodejs/iojs.org PR with "proof" of approval history already in place.
- If you'd have access to the https://github.com/nodejs/iojs.org repo (most contributors) then the best option is to run the commands against your own fork of https://github.com/nodejs/iojs.org, just like the normal requirement for posting any PR to https://github.com/nodejs/iojs.org -- this requirement would be the same for opening a PR whether or not we used subtrees.

In time, if we got enough i18n subtrees established, we could likely automate some of the importing to avoid the second PR.

**Comment by snostorm** [comment]
For the sake of speed I can also just do the subtree steps for now and work to help document them better after?

**Comment by yoannmoinet** [comment]
I'm ok with this @snostorm.
I don't see anything wrong if you document it and help us understand it :sweat_smile: 
Thanks for your help !
